src/Future.js

```diff
@@ -33,7 +33,7 @@ Future.prototype.ap = function(m) {
 
   return new Future(function(rej, res) {
     var applyFn, val;
-    var doReject = R.once(rej);
+    var doReject = rej != null ? R.once(rej) : function() {};
```

Comment:
Is `null` sneaking into the `rej` argument via some code elsewhere in this type for this occur, or is it due to `null` being given to `fork`? If the latter, I'd prefer to not encourage `null` being accepted as argument values here and recommend the no-op function that's defaulted to here be used as the argument when `fork` is called instead.

Perhaps an alternative would be to create a `forkResolved :: (a -> ()) -> ()` for this use case, which only accepts the resolved callback.
```
Future.prototype.forkResolved = function(res) {
  return this.fork(function noop(){}, res);
};
```